src/desktop/src/libs/SeedStore/Keychain.js

```diff
@@ -195,6 +195,40 @@ class Keychain {
         return decryptedVault;
     };
 
+    /**
+     * Unique seed check
+     * @param {array} Seed - Seed to check
+     * @returns {boolean} If Seed is unique
+     */
+    isUniqueSeed = async (seed) => {
+        const vault = await Electron.listKeychain();
+        if (!vault) {
+            throw new Error('Local storage not available');
+        }
+        try {
+            const accounts = vault.filter((acc) => acc.account !== ACC_MAIN && acc.account !== `${ACC_MAIN}-salt`);
+
+            for (let i = 0; i < accounts.length; i++) {
+                const account = accounts[i];
+
+                const vaultSeed = await decrypt(account.password, this.key);
+
+                if (
+                    vaultSeed.length === seed.length &&
```

Comment:
Doesn't this check make this one redundant? https://github.com/iotaledger/trinity-wallet/blob/f050c68d1d44ab1ec3adaa50f8b2310ce197124b/src/desktop/src/ui/views/onboarding/SeedVerify.js#L86